src/pages/[platform]/build-a-backend/storage/copy-files/index.mdx

```diff
@@ -66,7 +66,41 @@ Cross identity ID copying is only allowed if the destination path has the the ri
 
 </Callout>
 
-### More `copy` options
+Alternatively, specify a non-default bucket target by including `bucket` in both the source and destination objects. 
+
+See [Configure additional storage buckets](/[platform]/build-a-backend/storage/set-up-storage/#configure-additional-storage-buckets) for more information.
+
+```javascript
+import { copy } from 'aws-amplify/storage';
+
+const copyFile = async () => {
+  try {
+    const response = await copy({
+      source: {
+        path: 'album/2024/1.jpg',
+        // Alternatively, path: ({identityId}) => `album/{identityId}/1.jpg`
+        bucket: "userDefinedFriendlyBucketName"
```

Comment:
as for this similar to path, i would like to see in comment alternative version of the same

Reply:
i like it. i'll display the opposite alternative for destination below as well